test/integration/discordactions.test.js

```diff
@@ -237,6 +237,28 @@ describe("Discord actions", function () {
       expect(res.body).to.be.an("object");
       expect(res.body.message).to.equal("Role added successfully!");
     });
+    it("should create reson and pass it down to the bot", async function () {
```

Comment:
what's reson?

Reply:
The value to be passed in X-Audit-Log-Reason header

Reply:
there's typo I guess it will be reason